index.js

```diff
@@ -72,26 +74,41 @@ export default class Channel {
   /**
    * Call all listeners with specified params
    */
-  dispatch () {
+  dispatch (...args) {
     if (!this._mute) {
       this._listeners.forEach(listener => {
-        listener.callback.apply(listener.context, arguments);
+        listener.callback.apply(listener.context, args);
       });
+    } else if (this._accumulate) {
+      this._accumulatedEvents.push(args);
     }
   }
 
   /**
    * Mute channel
+   * @param {Object} [options]
+   * @param {Boolean} [options.accumulate] accumulate events and call listeners after .unmute()
    */
-  mute () {
+  mute (options = {}) {
     this._mute = true;
+    if (options.accumulate) {
+      this._accumulate = true;
+    } else {
+      this._accumulate = false;
+      this._accumulatedEvents = [];
```

Comment:
I think that it's not so evedent to clear accumulated events, when we, e.g. call twice:
```javascript
channel.mute({accumulate: true});
...
channel.mute({accumulate: true});
```
As for me in that case I would expect that nothing happens if we already accumulate events